### ISSUE TITLE
manual init bypassing freeze-up but initializes variables at origin

### ISSUE THREAD

**Issue by mc2922** [JuliaRobotics/Caesar.jl#371]
Re: issue [370](https://github.com/JuliaRobotics/Caesar.jl/issues/370)
See script: [testDynPtExManual](https://github.com/JuliaRobotics/Caesar.jl/blob/dynSAS/examples/marine/asv/kayaks/testDynPtExManual.jl)

Same fg that freezes, now added manualinit at line 70. Solve completing:
![image](https://user-images.githubusercontent.com/11624503/61165191-e7778280-a4d1-11e9-8487-1e023e88cab3.png)

but variables are initialized at 0:
![image](https://user-images.githubusercontent.com/11624503/61165195-f52d0800-a4d1-11e9-8628-5e684c929df4.png)
The sas-factor here is x5-x9

**Comment by dehann** [comment]
Hi, thanks for posting -- let me have a closer look.  My feeling is there is autoinit bug that is at least partially related to this "numerical" error bug.  Example is very helpful in finding and fixing, thanks.

**Comment by dehann** [comment]
HI @mc2922 , is it possible to plot the ground truth values perhaps please?

**Comment by mc2922** [comment]
Yup, Line 95 should be plotting ground truth? Let me check it

**Comment by dehann** [comment]
Update with latest IIF master, looks like it is working perhaps.  @mc2922 if you could please confirm these results?  I'm having a little trouble with the ground truth still -- anyway here is what I get now:
![Screenshot from 2019-08-02 02-25-05](https://user-images.githubusercontent.com/6412556/62349303-580c3080-b4cd-11e9-8dff-4569a56a587b.png)
![Screenshot from 2019-08-02 02-24-58](https://user-images.githubusercontent.com/6412556/62349307-5b072100-b4cd-11e9-9c3e-889df46e6acb.png)

**Comment by dehann** [comment]
I think the manual init is still failing, but autoinit is now performing better.  Here is manual init result:
![Screenshot from 2019-08-02 02-35-36](https://user-images.githubusercontent.com/6412556/62349618-4bd4a300-b4ce-11e9-9a0d-27fa012df492.png)

**Comment by mc2922** [comment]
Yes, freeze is resolved, now the tree-init completes with the following result: 
![image](https://user-images.githubusercontent.com/11624503/62351982-e1b30280-b4bb-11e9-9246-a10dd7e17060.png)

**Comment by mc2922** [comment]
Seeing the same issue with manual init, points are at the origin 
![image](https://user-images.githubusercontent.com/11624503/62352218-79185580-b4bc-11e9-90f3-0022a7b27754.png)

**Comment by dehann** [comment]
probably fixed, but will retest for v0.4.3
